stripe-core/src/main/java/com/stripe/android/core/strings/IdentifierResolvableString.kt

```diff
@@ -2,16 +2,22 @@ package com.stripe.android.core.strings
 
 import android.content.Context
 import androidx.annotation.StringRes
+import com.stripe.android.core.strings.transformations.TransformOperation
 import kotlinx.parcelize.Parcelize
 import kotlinx.parcelize.RawValue
 
 @Parcelize
 internal data class IdentifierResolvableString(
     @StringRes private val id: Int,
+    private val transformations: List<TransformOperation> = listOf(),
```

Comment:
```suggestion
    private val transformations: List<TransformOperation> = emptyList(),
```

Reply:
```
@kotlin.internal.InlineOnly
public inline fun <T> listOf(): List<T> = emptyList()
```

I kinda like using listOf instead of emptyList, it makes adding something down the road easier. So my default has basically just been to never use `emptyList`